### PR TITLE
Demo apps

### PR DESCRIPTION
**Pull request by infamousjoeg**
Adding demo apps to be used as the client in demonstrations concerning CyberArk Conjur.

* Kubernetes-Conjur-Demo Ruby Demo App (authn-k8s-ruby)
* pcf-vaultconjursync-ee Ruby Demo App (authn-pcf-ruby)
* Python-Flask Demo App (python-flask)

### CODE REVIEW

File: demo-apps/authn-pcf-ruby/Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'sinatra'
+gem 'conjur-api'
+gem 'conjur-cli'
```
Comment (jvanderhoof) [inline demo-apps/authn-pcf-ruby/Gemfile]: Do you need the CLI gem?

File: demo-apps/authn-pcf-ruby/.ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+ruby-2.4.1
```
Comment (jvanderhoof) [inline demo-apps/authn-pcf-ruby/.ruby-version]: Looks like the app container is Ruby 2.3.  This file isn't really needed. It's typically used for ruby version switching in RVM/RBEnv.

**Comment by infamousjoeg** [comment]
@jvanderhoof Point well received.  Do current commits match what you expect?

File: demo-apps/authn-pcf-ruby/Gemfile.lock [hunk 3]
```diff
@@ -0,0 +1,68 @@
+GEM
```
Comment (jvanderhoof) [inline demo-apps/authn-pcf-ruby/Gemfile.lock]: You'll need to commit the re-run `docker install` so the Gemfile.lock matches the Gemfile.

File: demo-apps/authn-pcf-ruby/secrets.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+DB_USERNAME_PROD: !var vault/PCF/P-APP-PCF-HELLOWORLD/Root/PROD_DBUser/username
+DB_PASSWORD_PROD: !var vault/PCF/P-APP-PCF-HELLOWORLD/Root/PROD_DBUser/password
+DB_USERNAME_DEV: !var vault/PCF/D-APP-PCF-HELLOWORLD/Root/DEV_DBUser/username
+DB_PASSWORD_DEV: !var vault/PCF/D-APP-PCF-HELLOWORLD/Root/DEV_DBUser/password
+STRIPE_API_KEY: !var pcf/app/stripe/private_key
```
Comment (jvanderhoof) [inline demo-apps/authn-pcf-ruby/secrets.yml]: Wouldn't these typically be in separate namespaces to control access?  Might be worth breaking them out into a `prod` & `dev` namespaces to align with best practices.

**Comment by jtuttle** [comment]
@infamousjoeg Are you planning on following up on this or should we close it for now? (we can keep the branch).

**Comment by infamousjoeg** [comment]
@jtuttle You can close and remove the branch.

**Pull request closed without merging**